src/Estimators/EstimatorManagerBase.cpp

```diff
@@ -97,7 +97,7 @@ EstimatorManagerBase::~EstimatorManagerBase()
 {
   delete_iter(Estimators.begin(), Estimators.end());
   delete_iter(RemoteData.begin(), RemoteData.end());
-  delete_iter(h5desc.begin(), h5desc.end());
```

Comment:
Could you say a bit how the leak happened? Is it because a pointe gets overwritten or it is a problem in the upper level classes? Since you looked at this so may have an answer.

Reply:
It gets allocated with `new` and in some instance it doesn't get deallocated and asan with lsan raises the issue in one of the tests.

Reply:
I'm wondering why it doesn't get de-allocated. "leak" means the pointer is no more accessible which is more than "not deallocated". So need to understand why this happens.